### DISCUSSION TITLE
Possibility to set setDimensionFilter / metricFilter

### DISCUSSION THREAD

**Discussion by bjora857** [gtmassey/Analytics#45] [Ideas]
Would be nice to be able to filter results on specific dimensions /filters

**Comment by Plytas** [comment]
Just wanted to let you know that I'm working on it and will submit PR soon. Here's a sneak peek:
<img width="818" alt="image" src="https://user-images.githubusercontent.com/17316322/209984070-f5f9bb39-ee95-48c5-a8f7-2d6fa5acd158.png">

**Comment by Plytas** [comment]
@bjora857 I think I've got all the logic implemented in `filters` branch. Still need to write tests and documentation before I could open a PR, but feel free to play around with it and let me know if you have any feedback.

**Comment by Plytas** [comment]
Opened a PR #49

**Comment by gtmassey** [comment]
PR #49 has been included in the v2.1.0 release!

**Reply by bjora857** [reply]
And it works perfect. Thank you so much! @Plytas and @gtmassey
